go-crud-psql/db/connect.go

```diff
@@ -0,0 +1,106 @@
+package db
+
+import (
+	"fmt"
+	"log"
+	"os"
+
+	"github.com/jinzhu/gorm"
+	_ "github.com/jinzhu/gorm/dialects/postgres"
+)
+
+type Store struct {
+	DB *gorm.DB
+}
+
+func Init() *Store {
+	user := goDotEnvVariable("DB_USER")
```

Comment:
Rather than reading the environment parameters here better to pass them as arguments to your constructor. This way it would be more testable.